site/en/blog/device-characteristics/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Chrome user device characteristics report
+description: >
+  Developers constantly wonder what kind of audience they are developing for. How much RAM do they have? What kind of Wi-fi are they on? A new report provides answers for a range of countries and platforms.
+date: 2021-09-17
+authors:
+  - jichen
+tags:
+  - device-characteristics
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/zIWz3bDI3mBRLki1VxMJ.jpeg'
+alt: >
+  An alternative text description of your hero or social image.
+---
+
+Developers are constantly wondering about what kind of audience they are
+developing for. Are they using high end devices with large RAM? What connection
+are they using (wifi, 2G...)? etc. The answers to such questions could carry
+much weight in many aspects of product development such as choosing the right
+technologies to render the content. This report provides timeline data to inform
+developers on these questions for a list of countries, like the US, Brazil,
+Japan, etc. Three platforms are included when data is available: Android,
+Windows, and ChromeOS.
+
+Looking at the data, there might be some interesting things for developers to
+think about. For example, in Japan on the Android platform, the majority of
+users have five to eight cores, and the prevalence is quite a bit higher than
+the US, and much higher than in NBU countries. Another example: if you look at
+the smallest display size in device pixels, in some countries like the US or
+Brazil, there's a clear trend that smallest display sizes (like 400-500) are
```

Comment:
This seems like a weird way to refer to display size. 400px in which dimension, x, y, diagonal? Looking at this [chart of iPhone resolutions](https://www.ios-resolution.com/), some legitimately big phones such as 12 pro max are 428 x 926. (The chart says that's logical pixels, but I think it's mislabeled.) Also, consider adding a link to an explainer of device pixels vs logical pixels.